ui/src/lib/gql/job/useQueries.ts

```
@@ -0,0 +1,90 @@
+  };
+};
+
+function toJobStatus(status: GraphqlJobStatus): JobStatus {
+  switch (status) {
+    case "RUNNING":
+      return "running";
+    case "COMPLETED":
+      return "completed";
+    case "FAILED":
+      return "failed";
+    case "PENDING":
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)